helm-charts/datacater/templates/pythonrunner.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ .Values.datacater.pythonrunner.name }}
+  labels:
+    {{- include "datacater.labelsPythonRunner" . | nindent 4 }}
+spec:
+  serviceName: {{ .Values.datacater.pythonrunner.serviceName }}
```

Comment:
This can not be variable, if it is not set to `pythonrunner-service.yaml#L4`. The service itself and this name have to be equal.

Reply:
I think fixing these names is better than keeping them variable. Our application has also assumptions and we can only change these values at startup time.